lib/index.js

```diff
@@ -1,7 +1,7 @@
 import '@babel/polyfill';
+import * as importedConstants from './constants';
 
-export * as constants from './constants';
-
```

Comment:
ES lint did not like this syntax, so I've imported it with a temporary name "importedConstants" and then re-exported it as "constants". Does this make sense?